### PR TITLE
Allow using webhook configurations as a library

### PR DESCRIPTION
**Pull request by pierDipi**
Moving webhook code to a library allows using webhook configurations
as a library and can be consumed by alternative implementations.

## Proposed Changes

- Allow using webhook configurations as a library

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
/lgtm
/approve

/hold
waiting for @matzew approval

**Comment by pierDipi** [comment]
/retest

Unrelated.

**Comment by matzew** [comment]
@pierDipi can you fix the linters ?

**Comment by pierDipi** [comment]
@matzew it has been done in this PR https://github.com/knative-sandbox/eventing-kafka/pull/128/files#diff-425bc5b8efe2b236d167797a6ec65d9f38755c102bd78f9548bd8b2f7e51af7cL61, however, if you want I can do it.

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
